### ISSUE TITLE
Program crash: assertion failure in qasciikey.cpp

### ISSUE THREAD

**Issue by bgottula** [seeing-things/track#236]
Program crashed with an error I've never seen before:

```
(venv) rgottula@nix:~/dev/track$ track --meridian-side=west --fuse star Deneb
In named star mode: 'Deneb'
Sensor fusion with camera enabled
No gamepads found.
ASSERT: "false" in file qasciikey.cpp, line 501
Aborted (core dumped)
```

This was particularly bad because it left the mount slewing at full speed until it hit the mount computer's axis limit. I had to power off the mount to prevent further motion.

**Comment by bgottula** [comment]
After a very brief search I found that the offending file is part of OpenCV and most likely is part of the GUI stuff. Not terribly surprising.

**Comment by bgottula** [comment]
Some additional search results for "qasciikey.cpp" suggest that [non-US keyboard layouts](https://stackoverflow.com/q/54777270) can cause problems, and some report that the problem occurs [over an ssh connection](https://forum.stereopi.com/viewtopic.php?t=910) but not locally. I think the most likely answer is that OpenCV's `waitKey()` [does not support Unicode characters](https://stackoverflow.com/a/54777380).

**Comment by bgottula** [comment]
The source file [qasciikey.cpp](https://github.com/qt/qtbase/blob/067b53864112c084587fa9a507eb4bde3d50a6e1/src/testlib/qasciikey.cpp) is actually part of the Qt backend, and the offending function is most likely [`keyToAscii`](https://github.com/qt/qtbase/blob/067b53864112c084587fa9a507eb4bde3d50a6e1/src/testlib/qasciikey.cpp#L203).

This is called from OpenCV's window_QT.cpp in `keyPressEvent()`. As of today, `nix` has version 4.5.3.56 of the `opencv-python` package installed in the `track` virtual environment. Here's the relevant line of code from OpenCV at the 4.5.3 tag as of today: 

https://github.com/opencv/opencv/blob/ad6e82942b37be8ee2c71c1d9bc7fe79cd16f7ab/modules/highgui/src/window_QT.cpp#L2168

**Comment by bgottula** [comment]
I was able to reproduce the crash using version 4.5.3.56 of `opencv-python`. I was trying to figure out how to input non-ascii characters in Linux Mint, and figured out how to set the right Alt key to be the "compose" key. This allows typing all kinds of special characters. However I discovered that simply pressing the compose key was enough to crash the program.

I also confirmed that after upgrading to `opencv-python-4.7.0.68` pressing the compose key no longer crashes the program. I went back and forth between versions a couple more times just to be certain.

I further found that the last version on PyPI with the bug is 4.5.5.64 (the last of the 4.5 series on PyPI) and the oldest version with the fix is 4.6.0.66 (the first of the 4.6 series on PyPI) which adds to the evidence that the commit I identified above is indeed the fix for this.
